tests/tests/test-fees/test-length-fees.ts

```diff
@@ -36,7 +36,7 @@ describeDevMoonbeam(
   (context) => {
     it("should have low balance transfer fees", async () => {
       const fee = await testBalanceTransfer(context);
-      expect(fee).to.equal(20958001520875n);
+      expect(fee).to.equal(79_359_001_520_875n);
```

Comment:
Is there a way to fetch this through a runtime API? I feel hardcoding this value is prone to having to change it every time

Reply:
It would be nice to avoid hard coding fee values everywhere, they break nearly all the time.

We have `verifyBlockFees`, which could be suitable here. Note that it calculates the fees by hand, as opposed to asking the node/runtime to verify it, which I think is important for some of these tests.

Maybe we need an equivalent for a single extrinsic...

Reply:
I thought there was a payment info runtime API that could return the expected fee, is that not suitable for this?